bundles/http/Citrix/Canvas-LMS/0b9f4444-56d6-4b4e-b108-5624645e71d2.js

```diff
@@ -1,449 +1,398 @@
+							user_login_id: enrollmentValue.user.login_id,
+							enrollment_state: enrollmentValue.enrollment_state,
+							user_id: enrollmentValue.user.id,
+							user_name: enrollmentValue.user.name
+						};
+						dataStore.save('enrollments', enrollmentBody);
+					}
 
-      if (announcementResponse.ok) {
-        let announcement = await announcementResponse.json();
-        for (const announcementValue of announcement) {
-          for (const enrollmentValue of enrollment) {
-            const announcementBody = {
-              id: announcementValue.id,
-              enrollme_id: enrollmentValue.id,
-              enrollme_course_id: enrollmentValue.course_id,
-              title: announcementValue.title,
-              message: announcementValue.message,
-              enrollme_user_login_id: enrollmentValue.user.login_id,
-              enrollme_role: enrollmentValue.role,
-              posted_at: announcementValue.posted_at,
-              delayed_post_at: announcementValue.delayed_post_at,
-              author_display_name: announcementValue.author.display_name,
-              url: announcementValue.url,
-            };
-            dataStore.save("announcements", announcementBody);
-          }
-        }
-      } else {
-        throw new Error(
-          `Announcement sync failed ${announcementResponse.status}:${announcementResponse.statusText}.`
-        );
-      }
-      tempEnrollmentLink.forEach((element) => {
-        const enrollmentLink = element.split("; ")[1];
-        if (enrollmentLink == 'rel="next"') {
-          enrollmentPageExists = true;
-        }
-      });
-    } while (enrollmentPageExists);
+					for (const announcementValue of announcement) {
+						for (const enrollmentValue of enrollment) {
+							const announcementBody = {
+								id: announcementValue.id,
+								enrollme_id: enrollmentValue.id,
+								enrollme_course_id: enrollmentValue.course_id,
+								title: announcementValue.title,
+								message: announcementValue.message,
+								enrollme_user_login_id: enrollmentValue.user.login_id,
+								enrollme_role: enrollmentValue.role,
+								posted_at: announcementValue.posted_at,
+								delayed_post_at: announcementValue.delayed_post_at,
+								author_display_name: announcementValue.author.display_name,
+								url: announcementValue.url
+							};
+							dataStore.save('announcements', announcementBody);
+						}
+					}
+				} else {
+					throw new Error(
+						`Enrollments sync failed ${enrollmentResponse.status}:${enrollmentResponse.statusText}.`
+					);
+				}
+				tempEnrollmentLink.forEach((element) => {
+					const enrollmentLink = element.split('; ')[1];
 
-    tempAnnouncementLink.forEach((element) => {
-      const announcementLink = element.split("; ")[1];
-      if (announcementLink == 'rel="next"') {
-        announcementPageExists = true;
-      }
-    });
-  } while (announcementPageExists);
+					if (enrollmentLink == 'rel="next"') {
```

Comment:
```suggestion
					if (enrollmentLink === 'rel="next"') {
```

Reply:
Mostly we use almost always `!==` and `===`.
You can read what's the difference i.e. [here](https://www.guru99.com/difference-equality-strict-operator-javascript.html) but the Internet is full of articles describing the difference.

But when you ask if it can have some problem exactly on this place, then answer is no. It would work with `==` and with `===` too. But it is just a good practice to use `===` when there is no good reason to use just `==`.
https://eslint.org/docs/rules/eqeqeq